rosapi_msgs/srv/GetParam.srv

```diff
@@ -1,4 +1,5 @@
 string name
-string default_value
```

Comment:
The PR says that the extra response fields are non-intrusive as you can just ignore them, but removing this default value probably will affect a few users.

I think it makes sense to remove it since you're just getting a parameter, and can handle falling back to a default on the client side, but just pointing out this is another breakage.

Reply:
AFAIK roslibjs does not use this feature but you're right that it still may affect some users, so I restored this field.